src/lib/project-saver-hoc.jsx

```diff
@@ -413,7 +383,8 @@ const ProjectSaverHOC = function (WrappedComponent) {
     ProjectSaverComponent.defaultProps = {
         autoSaveIntervalSecs: 120,
         onRemixing: () => {},
-        onSetProjectThumbnailer: () => {}
+        onSetProjectThumbnailer: () => {},
+        onUpdateProjectData: saveProjectToServer
```

Comment:
We should remove this eventually, although adding the default makes this PR seamless, so I think it's the way to go now.